frontend/src/app/modules/imports/services/data.service.ts

```diff
@@ -246,8 +265,22 @@ export class DataService {
     if (chartImg !== '') {
       formData.append('chart', chartImg);
     }
-    return this._http.post(`${this.urlApi}/export_pdf/${importId}`, formData, {
+    return this._http.post(`${this.getUrlApi()}/export_pdf/${importId}`, formData, {
       responseType: 'blob',
     });
   }
+
+  getDestinations(): Observable<Array<Destination>> {
+    return this._http.get<Array<Destination>>(
+      `${this.config.API_ENDPOINT}/import/destinations/`,
+      {}
+    );
+  }
+
+  getDestination(code): Observable<Destination> {
+    return this._http.get<Destination>(
```

Comment:
It would be beneficial to handle potential errors when making HTTP requests. This could be done by using RxJS operators such as catchError to handle any errors that may occur during the HTTP request and provide a more graceful error handling mechanism. [important]